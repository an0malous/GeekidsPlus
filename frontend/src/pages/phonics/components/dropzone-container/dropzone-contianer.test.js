import React from 'react';
import { shallow } from 'enzyme';
import DropzoneContainer from './dropzone-container.component';

describe('Dropzone Container', () => {
	let props;
	let component;

	beforeEach(() => {
		props = {
            currentWordLetters: ["g", "u", "m"],
		
			}
				const {currentWordLetters } = props
		component = shallow(<DropzoneContainer currentWordLetters={currentWordLetters}/>)
	});

	it('renders without crashing', () => {
		expect(component).toMatchSnapshot();
   });
});
