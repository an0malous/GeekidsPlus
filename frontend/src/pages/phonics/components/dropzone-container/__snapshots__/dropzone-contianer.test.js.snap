// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropzone Container renders without crashing 1`] = `
<Connect(Interact)>
  <Grid>
    <GridRow
      style={
        Object {
          "justifyContent": "space-evenly",
        }
      }
    >
      <Dropzone
        className="inner-dropzone"
        key="g"
        letter="g"
        style={
          Object {
            "border": "dotted white 2px",
            "color": "rgba(0,0,0,0)",
            "fontSize": "1.5rem",
            "padding": "25px",
          }
        }
      />
      <Dropzone
        className="inner-dropzone"
        key="u"
        letter="u"
        style={
          Object {
            "border": "dotted white 2px",
            "color": "rgba(0,0,0,0)",
            "fontSize": "1.5rem",
            "padding": "25px",
          }
        }
      />
      <Dropzone
        className="inner-dropzone"
        key="m"
        letter="m"
        style={
          Object {
            "border": "dotted white 2px",
            "color": "rgba(0,0,0,0)",
            "fontSize": "1.5rem",
            "padding": "25px",
          }
        }
      />
    </GridRow>
  </Grid>
</Connect(Interact)>
`;
